docs/Configuring-Pantheon/Networking.md

```diff
@@ -5,7 +5,40 @@ description: Pantheon networking is about P2P discovery and communication betwee
 
 Pantheon uses the network to find and connect to peers. 
 
-## Firewalls and Incoming Connections
+## Port Configuration 
+
+Ports must be exposed appropriately to enable communication. An example port configuration for a Pantheon node on AWS is: 
+                                     
+![Port Configuration](../images/PortConfiguration.png)
+
+### Peer Discovery  
```

Comment:
Would it be better to title this something like P2P Networking?  Since the text also talks about TCP connections which happen outside of discovery?

Reply:
Updated